packages/components/src/color-palette/test/index.tsx

```diff
@@ -20,6 +23,18 @@ function getWrappingPopoverElement( element: HTMLElement ) {
 	return element.closest( '.components-popover' );
 }
 
+const UnmockedColorPalette = () => {
+	const [ color, setColor ] = useState< string | undefined >( undefined );
+
+	return (
+		<ColorPalette
+			value={ color }
+			colors={ EXAMPLE_COLORS }
+			onChange={ ( newColor ) => setColor( newColor ) }
+		/>
+	);
+};
+
```

Comment:
I think `ControlledColorPalette` could be a better name here. Just for completeness sake, we could also decide to expose `onChange` in case we still wanted to inject a spy, as we did on other tests

```suggestion
const ControlledColorPalette = ( { onChange } ) => {
	const [ color, setColor ] = useState< string | undefined >( undefined );

	return (
		<ColorPalette
			value={ color }
			colors={ EXAMPLE_COLORS }
			onChange={ ( newColor ) => {
				setColor( newColor );
				onChange?.( newColor );
			} }
		/>
	);
};

```